deltachat-ios/Assets.xcassets/ic_launcher.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
You don't use the ic_launcher imageset from what I can see. I think it can be removed?!